endoreg_db/views/finding/finding.py

```
@@ -102,4 +102,11 @@ def morphology_classifications(self, request, pk=None):
             
             return Response(result)
         except Exception as e:
-            return Response({'error': str(e)}, status=500)
\ No newline at end of file
+            return Response({'error': str(e)}, status=500)
+      
+    @action(detail=False, methods=['get'], url_path='by-id/(?P<finding_id>[^/.]+)')
+    def get_finding_by_id(self, request, finding_id=None):
```

Ruff (0.12.2)

Unused method argument: `request`

(ARG002)